metrics/manager/manager_test.go

```diff
@@ -23,15 +23,15 @@ import (
 )
 
 func TestFlow(t *testing.T) {
-	source := util.NewDummyMetricsSource("src", time.Millisecond)
+	source := util.NewDummyMetricsSource("src", 500*time.Millisecond)
 	sink := util.NewDummySink("sink", time.Millisecond)
-	processor := util.NewDummyDataProcessor(time.Millisecond)
+	processor := util.NewDummyDataProcessor(500 * time.Millisecond)
 
-	manager, _ := NewManager(source, []core.DataProcessor{processor}, sink, time.Second, time.Millisecond, 1)
+	manager, _ := NewManager(source, []core.DataProcessor{processor}, sink, MinMetricResolution, time.Millisecond, 1)
 	manager.Start()
 
-	// 4-5 cycles
-	time.Sleep(time.Millisecond * 4500)
+	// 4.5 * MinMetricResolution, about 4-5 cycles
```

Comment:
now that I think about it, I don't see much reason to actually respect the minimum metric resolution with the dummy sink -- this is just going to make the tests take *way* longer to run.